src/popup/pages/RestoreAccount/index.js

```diff
@@ -53,7 +53,6 @@ class RestoreAccount extends React.Component {
     sendMsg({
       action: WALLET_NEW_HD_ACCOUNT,
       payload: {
-        pwd: this.state.password,
```

Comment:
will this affect #11?

Reply:
Nope, this was just `undefined` before

Reply:
oh ok, thanks for the analysis.

we'll have to see separately about the UI code for the recovery in #11.